e2e/select/select.chromium.e2e-spec.ts

```diff
@@ -0,0 +1,16 @@
+import {SelectPO} from '@agnos-ui/page-objects';
+import {expect, test} from '../fixture';
+import {analyze} from '../utils';
+
+test.describe('Accessibility', () => {
+	test('Opened select', async ({page, browserName}) => {
+		if (browserName === 'chromium') {
+			await page.goto('#/select/default');
+			const selectPO = new SelectPO(page);
+			const locatorInput = selectPO.locatorInput;
+			await locatorInput.fill('a');
+			await locatorInput.press('Enter');
+			expect((await analyze(page, page.url())).violations).toEqual([]);
```

Comment:
Note that in the main accessibility test, we test for both light and dark modes.
I wonder if we should do the same here...